lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -221,8 +215,9 @@ def test_accessing_user_data_with_invalid_token(
             self.api_method, ParamType.USER, None
         )
         if not userp:
+            # API does not have a USER parameter.
             pytest.skip(
-                "skipping " + self.test_accessing_user_data_with_invalid_token.__name__
+                f"skipping {self.test_accessing_user_data_with_invalid_token.__name__}"
             )
```

Comment:
I don't think this change accomplished much (it looks like you removed four characters and added four characters...), but it does bring up a question:  does having the word "skipping" in the text here actually provide any value?  Don't we know from context (i.e., the effect of the `skip()` call) that something was skipped, and what we're interested in reporting is what and/or why?

Reply:
It is not even reported until you go super verbose, so this is just glorified documentation.

The change came from first removing all the skips and then adding them back without using `+` string concatenation.

Reply:
I stand corrected.

Adding those comments to the skip message means they show up in the report.  See the latest update.